src/posts/blogs/2020/2020-03-11-deploying-linkerd-with-meshery/index.mdx

```diff
@@ -0,0 +1,141 @@
+</div>
+<div>
+<h5>Meshery Adapters</h5>
+    <table class="table-adapters">
+        <thead class="hidden">
+            <th>Status</th>
+            <th>Adapter</th>
+        </thead>
+        <tbody>
+        <tr>
+            <td rowspan="7" class="stable-adapters">stable</td>
+        </tr>
+        <tr>
+            <td><a href="https://github.com/layer5io/meshery-istio">
+                <img src={istio} alt='Istio Service Mesh adapter' class="adapter-logo" />Meshery adapter for Istio</a> 
+            </td>
+        </tr>
+        <tr>
+            <td><a href="https://github.com/layer5io/meshery-linkerd">
+                <img src={linkerd} alt='Linkerd' class="adapter-logo" />Meshery adapter for Linkerd</a> 
+            </td>
+        </tr>
+        <tr>
+            <td><a href="https://github.com/layer5io/meshery-consul">
+                <img src={consul} alt='Consul Connect' class="adapter-logo" />Meshery adapter for Consul</a>
+            </td>
+        </tr>
+        <tr>
+            <td><a href="https://github.com/layer5io/meshery-octarine">
+                <img src={octarine} alt='Octarine Service Mesh' class="adapter-logo" />Meshery adapter for Octarine</a> 
+            </td>
+        </tr>
+        <tr>
+            <td><a href="https://github.com/layer5io/meshery-nsm">
+                <img src={nsm} alt='Network Mesh' class="adapter-logo" />Meshery adapter for Network Service Mesh</a>
+            </td>
+        </tr>
+        <tr><td class="stable-adapters"></td></tr>
+        <tr>
+            <td rowspan="2" class="beta-adapters">beta</td>
+            <td><a href="https://github.com/layer5io/meshery-cpx">
+                <img src='https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQksHj15DkID308qQw3cmkQrRULPxyzbVquSZVev-9dj1L6sPs-rQ&s' alt='Citrix CPX Service Mesh' class="adapter-logo" />Meshery adapter for Citrix CPX</a>
+            </td>
+        </tr>
+        <tr><td class="beta-adapters"></td></tr>
+        <tr>
+            <td rowspan="6" class="alpha-adapters">alpha</td>
+        </tr>
+        <tr>
+            <td><a href="https://github.com/layer5io/meshery-maesh">
+                <img src='https://raw.githubusercontent.com/containous/maesh/master/docs/content/assets/img/maesh.png' alt='Maesh Service Mesh' class="adapter-logo" />Meshery adapter for Maesh</a>
+            </td>
+        </tr>
+        <tr>
+            <td><a href="https://github.com/layer5io/meshery-app-mesh">
+                <img src={awsappmesh} alt='AWS App Mesh Service Mesh' class="adapter-logo" />Meshery adapter for App Mesh </a>
+            </td>
+        </tr>
+        <tr>
+            <td><a href="https://github.com/layer5io/meshery-kuma">
+                <img src='https://pbs.twimg.com/profile_images/1171109603158843392/bV2M7l39_400x400.jpg' alt='Kuma Service Mesh' class="adapter-logo" />Meshery adapter for Kuma</a>
```

Comment:
This image is not available.

Reply:
This image is not available in the current site as well.

Reply:
[Here]( https://layer5.io/assets/images/landscape/Kuma.svg) you go. We should also open up an issue for the existing site I think

Reply:
Cool, I will do it.